inspirehep/dojson/hep/fields/bd20x24x.py

```diff
@@ -32,7 +32,6 @@
 
 
 @hep.over('title_translations', '^242[10_][0_]')
```

Comment:
We can have everyhing in one field, and use this guy: https://github.com/jprante/elasticsearch-langdetect.